src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -177,7 +252,8 @@ public StarttlsOption getStarttls() {
    *
    * Either NONE, OPTIONAL or REQUIRED
    *
-   * @param starttls (default is OPTIONAL)
+   * @param starttls
+   *          (default is OPTIONAL)
    * @return a reference to this, so the API can be used fluently
    */
   public MailConfig setStarttls(StarttlsOption starttls) {
```

Comment:
the command is called "STARTTLS" in all-caps, so I ended up calling the member variable starttls and the getter got like this, we can change that if you like